### PR TITLE
update libs, target sdk 33

### LINKED ISSUE #26: Update firebase libs in library
Hi,
Please update the libraries you use, we have to write all the versions of the libraries we use ourselves to avoid version conflicts. For example:
`Duplicate class com.google.firebase.iid.FirebaseInstanceIdReceiver found in modules jetified-firebase-iid-20.1.5-runtime (com.google.firebase:firebase-iid:20.1.5) and jetified-firebase-messaging-23.1.0-runtime (com.google.firebase:firebase-messaging:23.1.0)`

### PR DESCRIPTION
**Pull request by ExNDY**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 # moko
-mokoCrashReportingVersion = "0.3.0"
+mokoCrashReportingVersion = "0.3.1"
 
 
 [libraries]
 # android
 appCompat = { module = "androidx.appcompat:appcompat", version.ref = "androidAppCompatVersion" }
-firebaseCore = { module = "com.google.firebase:firebase-core", version.ref = "firebaseCoreVersion" }
+firebaseAnalytics = { module = "com.google.firebase:firebase-analytics", version.ref = "firebaseAnalyticsVersion" }
```
Comment (Alex009) [inline gradle/libs.versions.toml:20]: why we should use analytics instead of core?

Comment (ExNDY) [reply gradle/libs.versions.toml:20]: https://firebase.google.com/support/release-notes/android

Release from 12 October, include recommendation for migrate on firebase-analytics from firebase-core. I thought replace used libraries on ktx versions, but im don't know will compatibility be maintained.
If u think what my solution include mistake with this library, i should replace on core.

Comment (Alex009) [reply gradle/libs.versions.toml:20]: ok

**Review by Alex009 — CHANGES_REQUESTED** [review]
CI should be fixed

Manifest merger failed : uses-sdk:minSdkVersion 16 cannot be smaller than version 19 declared in library [com.google.firebase:firebase-analytics:21.1.1] /Users/runner/.gradle/caches/transforms-3/087771e3bb90307ccd11affb28f96c0f/transformed/firebase-analytics-21.1.1/AndroidManifest.xml as the library might be using APIs not available in 16
[855](https://github.com/icerockdev/moko-crash-reporting/actions/runs/3350791421/jobs/5553578705#step:5:856)
  	Suggestion: use a compatible library with a minSdk of at most 16,
[856](https://github.com/icerockdev/moko-crash-reporting/actions/runs/3350791421/jobs/5553578705#step:5:857)
  		or increase this project's minSdk version to at least 19,
[857](https://github.com/icerockdev/moko-crash-reporting/actions/runs/3350791421/jobs/5553578705#step:5:858)
  		or use tools:overrideLibrary="com.google.firebase.firebase_analytics" to force usage (may lead to runtime failures)

**Review by Alex009 — APPROVED** [review]

**Pull request merged**